### DISCUSSION TITLE
Bulk export of findings into CSV

### DISCUSSION THREAD

**Discussion by silent6trinity** [Syslifters/sysreptor#40] [General]
First off - this is a fantastic tool, you guys really did a great job simplifying the report process.

One thing that would be nice to have, would be the potential to bulk export the findings into a CSV/.XLS format.

**Comment by aronmolnar** [comment]
Thank you!!

An export into CSV/XLS would not include images.
If you export a project, you receive a tar.gz that you can extract. You find a JSON structure there containing your findings and also the images.

**Comment by richardschwabe** [comment]
Here is an example:

https://github.com/richardschwabe/sysreptor_xlsx_findings

Hope it helps

**Reply by aronmolnar** [reply]
Very nice, thank you!

**Reply by silent6trinity** [reply]
This is great, exactly the functionality I was looking for - would be super nice to have it be a function built into the WebUI for potential non-technical folk. But thanks @richardschwabe !

**Comment by aronmolnar** [comment]
You can now do this via our CLI tool.

```
pip3 install reptor
reptor conf
reptor projectfindings --format csv  # add custom fieldnames: --fieldnames title,cvss
```

**Reply by silent6trinity** [reply]
I have tested this on my instances and they work very well. Thank you so much for doing this - its going to be a huge help.

Do you plan to make _all_ field names queryable in the future? Like, the "non-standard" ones - like from the *report* fields? Like if I wanted to query "Executive Summary" for example?

**Reply by aronmolnar** [reply]
It should actually be possible for all fieldnames. You can reference the by your finding field ID (that you also use in HTML/Vue).
If you have issues with more complex field types, just write and we'll find a solution.
